package.json

```diff
@@ -33,6 +33,7 @@
   "dependencies": {
     "@kiwicom/orbit-design-tokens": "^0.2.5",
     "react-datepicker": "2.0.0",
+    "react-native-multi-slider": "https://github.com/lukewalczak/react-native-multi-slider.git#feat/block-overlap",
```

Comment:
Great work! We will need to create an issue to take over that fork.

Reply:
I can transfer ownership to you or you can create a fork from my fork